user_guide/access_permissions.rst

```diff
@@ -0,0 +1,144 @@
+.. _user-guide-user-management-permissions:
```

Comment:
Please provide reference to https://github.com/orocrm/documentation/blob/master/book/security.rst and make sure that content plays nicely together.